package.json

```diff
@@ -39,6 +39,7 @@
     "karma-mocha": "~2.0.1",
     "karma-spec-reporter": "~0.0.32",
     "mocha": "~8.3.0",
+    "playwright": "^1.9.1",
```

Comment:
```suggestion
    "playwright": "~1.9.1",
```

Reply:
make sure to regenerate the package-lock after accepting this suggestion

Reply:
Also roll this dependency upgrade into your PR too: https://github.com/rooseveltframework/teddy/pull/472